packages/perseus/src/widgets/interactive-graphs/reducer/interactive-graph-state.test.ts

```diff
@@ -109,6 +109,21 @@ describe("initializeGraphState for line graphs", () => {
             ],
         ]);
     });
+
+    it("ignores any provided coords", () => {
+        // This is a characterization test. I don't know if it's desired
+        // behavior, but it's the existing behavior.
```

Comment:
😢  I'm pretty sure this is a byproduct of the way that our editors work. They serialize the props passed into the graph component and don't always strip out unnecessary data. Thus we have `.graph.coords` that is populated, when we really only care about `.correct.coords` (the rubric). 

We also automatically derive the starting location of the movable lines and so that's why the coords in the data are ignored.